Microsoft.Toolkit.Uwp.UI.Controls.Design/TabbedCommandBarMetadata.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp.UI.Controls.Design.Common;
+using Microsoft.Windows.Design;
+using Microsoft.Windows.Design.Features;
+using Microsoft.Windows.Design.Metadata;
+using Microsoft.Windows.Design.Model;
+using Microsoft.Windows.Design.PropertyEditing;
+using System.ComponentModel;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
+{
+
+    internal class TabbedCommandBarMetadata : AttributeTableBuilder
+    {
+        public TabbedCommandBarMetadata()
+            : base()
+        {
+            AddCallback(typeof(TabbedCommandBar),
+                b =>
+                {
+                    //b.AddCustomAttributes(nameof(TabbedCommandBar.SomePropertyHere),
+                    //    new CategoryAttribute(Properties.Resources.CategoryCommon)
+                    //    );
```

Comment:
Is this still needed? if not we should delete

Reply:
I'm actually not sure what this class is for. Do you have any insights?

Reply:
Oh, I'm not sure. Did you add this? or something else added this file?

Reply:
Every control in the Toolkit appears to have a metadata class like this. Maybe it has something to do with the 'Properties' tab in the sample app?

Reply:
The Metadata class is to help organize properties within the Visual Studio Properties panel for the Designer. You should follow the pattern of other controls here and split out any custom properties into the proper categories you'd think they'd fit within the VS Designer.